### PR TITLE
Publisher Updates

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

**Comment by clintonb** [comment]
Dependent on https://github.com/edx/course-discovery/pull/1143

**Comment by clintonb** [comment]
I have removed a couple test cases because it's not clear that they provide any value. Sorting is handled by a third-party library. There is no need to test it to such extreme levels.

File: course_discovery/apps/publisher/templates/publisher/email/studio_instance_needed.txt [hunk 1]
```diff
@@ -1,18 +0,0 @@
-{% load i18n %}
```
Comment (schenedx) [inline course_discovery/apps/publisher/templates/publisher/email/studio_instance_needed.txt:1]: Don't we still need this message for ASU? Or for those organization which do not want auto studio instance creation?

**Review by schenedx — COMMENTED** [review]
Just one question, do we want to delete those studio emails?

File: course_discovery/apps/publisher/templates/publisher/email/studio_instance_needed.txt (hunk 1 above)
Comment (clintonb) [reply course_discovery/apps/publisher/templates/publisher/email/studio_instance_needed.txt:1]: This template was never used by any code. It seems to be have been created unnecessarily.

**Pull request merged**